Assignment/training2/src/task3/Features/Home.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import './Home.css';
+
+
+const checkNumber = (input) =>
+{
+    if(isNaN(input))
+    {
+        return String;
+    }
+    else
+    {
+        return Number;
+    }
+}
+
+
+function Home3()
+{
+    const [content, setContent] = useState('');
+    const [alert, setAlert] = useState('');
+    const [countDown, setCountDown] = useState(0);
+    const [hiddenStop, setHiddenStop] = useState(true);
+    const [hiddenPlay, setHiddenPlay] = useState(true);
+    const [loop, setLoop] = useState();
+
+    if(countDown === 0) { clearInterval(loop)}
+    const counter = () =>{
```

Comment:
@lehuy1012 so much problem and the code is not clean, try to apply useEffect (lifecycle in main purpose of this training number)

Reply:
fixed